src/inventory/weapons/InstantWeapon.cpp

```diff
@@ -66,9 +66,12 @@ bool InstantWeapon::fire(Movable& movable) {
     }
     logv(3, "InstantWeapon::fire()\n");
 
+
+    const double deviation = rand() % accuracy + (accuracy / 2);
```

Comment:
I'm glad you're adding deviation for shots
But doesn't the way you have this set up mean that all shots will only deviate clockwise instead of equally to both sides?
It seems like accuracy while being an int, will only be set to positive
And since rand returns a positive number, your deviation will only ever be positive
Seems like you need some adjustment to make deviation go equally to both sides